packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -147,3 +147,27 @@ export const nextDescendingStepTime = (liveSchedule, nextSchedule, now) => {
   return expectedNext;
 };
 harden(nextDescendingStepTime);
+
+/**
+ *
+ * @param {Timestamp} time
+ * @param {import('./scheduler.js').Schedule} schedule
+ * @returns {'before' | 'during' | 'endExactly' | 'after'}
+ */
+export const timeVsSchedule = (time, schedule) => {
+  const { startTime, endTime } = schedule;
+  assertAllDefined({ startTime, endTime });
+
+  if (TimeMath.compareAbs(time, schedule.startTime) < 0) {
+    return 'before';
+  }
+  if (TimeMath.compareAbs(time, schedule.endTime) < 0) {
```

Comment:
Is startTime..endTime supposed to be an inclusive..exclusive interval? That is what I'd expect, in which case this code is correct. Otherwise, perhaps one of these `<`s should be a `<=`?

Reply:
`during` is for the inclusive..exclusive interval. `endExactly` covers the remainder of the fully inclusive interval.